src/pretalx/agenda/templates/agenda/talks.html

```diff
@@ -7,7 +7,7 @@
 {% block agenda_content %}
     {% include "agenda/header_row.html" %}
 <p></p>
-    {% for talk in talks %}
+    {% for talk in talks_distinct %}
         <section>
```

Comment:
Talks overview page -
only showing every Topic once - the details (times and rooms) are on the 'Talk' page